bool kotak
bool silang
bool bulat
bool segitiga
bool L1
bool R1
bool L2
bool R2
bool share
bool options
bool L3_B
bool R3_B
bool PS
bool TP
bool atas
bool bawah
bool kiri
bool kanan
float32 LX
float32 LY
float32 RX
float32 RY
float32 L2A
float32 R2A